File: PHP Excel Examples/WorksheetsApiDemo/FreezePanes/setFreezePanes.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "SetFreezePanes.xlsx";
$storage = null;
$folder = "input";
$sheetName = "Sheet2";
$freezedRows = 3;
$freezedColumns = 3;
$result = $apiInstance->setFreezePanes($name, $sheetName,$freezedRows,$freezedColumns,$folder, $storage);

File: Python Excel Examples/WorksheetsApiDemo/freeze_panes/setFreezePanes.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "SetFreezePanes.xlsx"
storage = ""
folder = "ExcelDocument"
sheet_name = "Sheet1"
freezedRows = 3
freezedColumns = 3
api.set_freeze_panes(name, sheet_name, freezedRows, freezedColumns, storage=storage, folder=folder)